pkg/collector/reconcile/configmap_test.go

```diff
@@ -103,8 +103,10 @@ receivers:
         metrics_path: /metrics
         scheme: http
         follow_redirects: true
+        enable_http2: true
```

Comment:
what are these additions for? new defaults?

Reply:
Probably, these changes are coming from the update of `github.com/prometheus`

Reply:
@jaronoff97 any concerns or objections to merge this?

Reply:
nope, should be okay!